zigpy_znp/uart.py

```diff
@@ -238,18 +238,18 @@ async def connect(config: conf.ConfigType, api, *, toggle_rts=True) -> ZnpMtProt
     # Skips the bootloader on slaesh's CC2652R USB stick
     if toggle_rts:
         LOGGER.debug("Toggling RTS/CTS to skip CC2652R bootloader")
-        transport.serial.dsrdtr = False
-        transport.serial.rtscts = False
+        transport.serial.dtr = False
+        transport.serial.rts = False
 
         await asyncio.sleep(RTS_TOGGLE_DELAY)
 
-        transport.serial.dsrdtr = False
-        transport.serial.rtscts = True
+        transport.serial.dtr = False
+        transport.serial.rts = True
 
         await asyncio.sleep(RTS_TOGGLE_DELAY)
 
-        transport.serial.dsrdtr = False
-        transport.serial.rtscts = False
+        transport.serial.dtr = False
+        transport.serial.rts = False
```

Comment:
To make unit tests pass, you also have to edit the property names used in `test_no_toggle_rts` and `test_toggle_rts` at the bottom of `tests/test_uart.py`.

Reply:
Changed, these tests now pass.